### PR TITLE
Import fixing

### PR DESCRIPTION
**Pull request by stevevanhooser**

### CODE REVIEW

**Comment by stevevanhooser** [comment]
Comments as I'm learning the new framework:

`importNetworkFromTensorFlow` appears to only be able to create its namespace in the present working directory; it would be nice if the function could save that namespace anywhere. I should post this on some MathWorks forum.

**Comment by stevevanhooser** [comment]
@ramnarak : Question: `importNetworkFromTensorFlow` creates a namespace. Is it used beyond the import itself? It doesn't seem to have an "open" method or anything like that that allows it to be read from the namespace more quickly. I was considering having our internal code delete the namespace that is made if it has no function after the import.

Thanks
Steve

**Comment by stevevanhooser** [comment]
@ramnarak : Ah, I answered my own question. Yes, the namespace is critical to the use of the network. So we will need to make sure it has a good name that doesn't conflict with other networks and put it on the path.

**Comment by stevevanhooser** [comment]
Will close #43

**Comment by stevevanhooser** [comment]
@vijayiyer05 Do you want to do a quick review on this? I can also merge it without review if you'd prefer. Thanks!

File: +deepinterp/+internal/openTensorFlowNetwork.m [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+net_namespace = namespaceName;
+currDir = pwd;
+
+did_it_fail = false;
+
+try,
+   cd(tfnet_path);
+   net = importNetworkFromTensorFlow(modelFolder,Namespace=namespaceName);
+catch,
+   did_it_fail = true;
```
Comment (vijayiyer05) [inline +deepinterp/+internal/openTensorFlowNetwork.m:31]: I wonder if the following idiom woudl be more lean: 

> catch ME
>       ME.rethrow()
> end

**Review by vijayiyer05 — APPROVED** [review]
I've put in a couple of suggestions for this review. 

Bigger picture: I think if it's working & sensible in the maintainers' hands, it makes sense to merge; and then we can do another pass on file/folder org, code elegance,  and usability.

File: +deepinterp/+internal/openTensorFlowNetwork.m (hunk 1 above)
Comment (stevevanhooser) [reply +deepinterp/+internal/openTensorFlowNetwork.m:31]: `importNetworkFromTensorFlow` only saves its namespace to the local directory. `openTensorFlowNetwork` is written that way to make sure we always change back to the user's current directory regardless of whether the import operation succeeds or fails.

File: pretrainedModels/pretrained.json [hunk 2]
```diff
@@ -2,13 +2,13 @@
     {
         "name":"TP-Ai93-450",
         "filename":"2019_09_11_23_32_unet_single_1024_mean_absolute_error_Ai93-0450.h5",
```
[the remark this answers is not in the document]
Comment (stevevanhooser) [reply pretrainedModels/pretrained.json:4]: not needed anymore, true! thanks!

**Pull request merged**